react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -0,0 +1,41 @@
+# Next.js. Server Side Rendering
+
+## Technical requirements
+
+1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr-pages-api".
+2. Next.js Pages API Integration:

```

Comment:
Not really sure about my requirements explanation. Want to ask reviewers to provide a piece of advice here.